tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -218,52 +208,6 @@ module.exports = {
       await this.clickRow(resource)
       return this.api.page.FilesPageElement.appSideBar()
     },
-    /**
-     * opens file-actions menu for given resource
-     *
-     * @param {string} resource name
-     * @param {string} resource type (file|folder)
-     *
-     * @returns {*}
-     */
-    openFileActionsMenu: function (resource, elementType = 'file') {
-      const fileActionsBtnSelector = this.getFileActionBtnSelector(resource, elementType)
-      this
-        .useXpath()
-        .waitForElementVisible(fileActionsBtnSelector)
-        .click(fileActionsBtnSelector)
-        .useCss()
-      return this.api.page.FilesPageElement.fileActionsMenu()
-    },
-
-    /**
-     *
-     * @param {string} path
-     */
-    markAsFavorite: async function (path) {
-      await this.waitForFileVisible(path)
-
-      const favoriteIconButton = this.getFileRowSelectorByFileName(path) +
-        this.elements.notMarkedFavoriteInFileRow.selector
-
-      await this.initAjaxCounters()
-        .useXpath()
-        .click(favoriteIconButton)
-        .waitForOutstandingAjaxCalls()
-        .useCss()
-
-      return this
-    },
-    /**
-     *
-     */
-    checkAllFiles: function () {
-      return this.initAjaxCounters()
-        .waitForElementVisible('@filesTable')
-        .waitForElementVisible('@checkBoxAllFiles')
-        .click('@checkBoxAllFiles')
-    },
-
```

Comment:
`check all files` is not part of file-row, IMO there is no need to move this function